### DISCUSSION TITLE
Another idea... Squad followers

### DISCUSSION THREAD

**Discussion by Digitalinsite** [Joe91/fun-bots#266] [Ideas]
Me again. Just another idea I thought of (probably already been mentioned I'm guessing...) - adding "follow me" to the Bot Command menu. Not sure if it can be done and is in the scope of the AI at all but thought it would be cool if you could issue a "Follow me" to a nearby teammate bot (or bots) and they basically follow you in about a 5m radius or something like that. Even better if they get in and out of vehicles with you. I reckon it would make some really cool AI squad gameplay with the bots. Leading a fireteam around the battlefield... 

Thanks again for the mod - I dug into the registry config file and changed the fire rocket variable to 80% and man, It's crazy fun now! Bots firing off RPG's left right and center. Lovin it.

**Comment by joao-vitor-souza** [comment]
@Digitalinsite, this idea was discussed in the [discord channel](https://discord.com/invite/FKamccAEqz). Basically, it was said that bots walk over pre-defined paths, so they won't be able to follow you, except if you're running on such path. @Joe91 can give you a better explanation.

**Comment by Joe91** [comment]
Yeah, kind of. But I thought a little more about that:
In the zombie/knife -mode the bots already follow a player they attack. That's done by kind of creating a new temporary trace on the fly.
This should also be possible in that case but requires quite a bit of extra code and handling also for situations when the player stops or so. And they will just walk in one line behind the player. But that would be possible with the current system ...
